src/barril/units/_tests/test_posc.py

```diff
@@ -378,3 +378,21 @@ def testFluidGasConcentration(unit_database_posc):
     assert approx(abs(q.ConvertScalarValue(1, "ppm") - 333.33), 7) == 0
     assert approx(abs(q.ConvertScalarValue(1, "%") - 0.033333), 7) == 0
     assert approx(abs(q.ConvertScalarValue(1, "Euc") - 0.00033333), 7) == 0
+
+
+def testSpringDashpotUnits(unit_database_posc):
+    """
+    Units used to define Spring-Dashpot movements
+    """
+    
+    q = ObtainQuantity("Ns/m")
+    assert "force per velocity" == q.GetQuantityType()
```

Comment:
Please write "assert obtained == expected"

Reply:
Yup, I found it weird to write like this, but the test right above this one was doing it like this and I thought I should keep consistency.

Reply:
Oh I didn't notice. I agree, let's keep it like this for now, some time we should update the entire file to avoid java-like asserts. 😁